tests/test-map.c

```diff
@@ -9,7 +9,23 @@ static int compare_fn_int(void *x, void *y) {
 	return 0;
 }
 
-void test_map(void) {
+bool predicate_fn_one(void *key) {
+	if (*(int *)key == 20) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+bool predicate_fn_two(void *key) {
+	if (*(int *)key == 30) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+void main(void) {
```

Comment:
This should be `void test_main(void)` because the `main()` is being called here: 
https://github.com/harsath/cap-containers/blob/master/tests/test-main.c#L14

Since all the test's implementation functions are called from `test-main.c`, it is where we need to have the `main()` function. Not on the test files.

As you can see, we are calling the `test_map` from the `test-main.c` here:
https://github.com/harsath/cap-containers/blob/master/tests/test-main.c#L22

Please change it back to `test_map` as before.